<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLevelsScene</class>
 <widget class="QWidget" name="GameLevelsScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1459</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noteworthy</family>
    </font>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>800</width>
     <height>620</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ShowCards">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>0</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noteworthy</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ваши карты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Shop">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>90</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noteworthy</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Магазин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>-40</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noteworthy</family>
     <pointsize>89</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Карта</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noteworthy</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noteworthy'; font-size:24pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Это карточная рогалик игра. &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;	Сейчас вы находитесь на карте уровней, в которой вы можете кликнув по &lt;span style=&quot; color:#c23ae2;&quot;&gt;Фиолетовому&lt;/span&gt; кружочку попасть к противнику на поединок, а &lt;span style=&quot; color:#ec3327;&quot;&gt;Красным&lt;/span&gt; пока недоступные вам уровни. Пройденный уровень обозначен &lt;span style=&quot; color:#002af4;&quot;&gt;Синим&lt;/span&gt; цветом, вернуться на него нельзя. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="PixMaps.qrc">:/new/prefix1/prefix1/BackGroundForGameLevelScene.jpeg</pixmap>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>pushButton_Back</zorder>
  <zorder>graphicsView</zorder>
  <zorder>pushButton_ShowCards</zorder>
  <zorder>pushButton_Shop</zorder>
  <zorder>label</zorder>
  <zorder>textBrowser</zorder>
 </widget>
 <resources>
  <include location="PixMaps.qrc"/>
 </resources>
 <connections/>
</ui>
